pkgs/by-name/ca/caper/package.nix

```
@@ -0,0 +1,60 @@
+{
+  stdenv,
+  lib,
+  ocaml,
+  ocamlPackages,
+  gnum4,
+  fetchFromGitLab,
+}:
+stdenv.mkDerivation rec {
+  pname = "caper";
+  version = "0.9";
+
+  src = fetchFromGitLab {
+    owner = "niksu";
+    repo = "caper";
+    rev = "v${version}";
+    hash = "sha256-TSryjz0NrGdkc+6vmfBqsuVpV3N9FvteTFsVqpUcm0w=";
+  };
+
+  nativeBuildInputs = [ ocaml ];
+
+  buildInputs = [
+    ocamlPackages.ocamlbuild
```

```suggestion
  nativeBuildInputs = [
    ocaml
    ocamlPackages.ocamlbuild
  ];

  buildInputs = [
```
It is used during build, not runtime.